### ISSUE TITLE
Object2Vec - validation loss increases while training loss decreases

### ISSUE THREAD

**Issue by tripuranenis** [aws/sagemaker-python-sdk#591]
Please fill out the form below.

### System Information
- Object2Vec SageMaker algorithm

### Describe the problem
I am trying to use Object2Vec SageMaker algorithm to predict ratings of music instruments.
Dataset is downloaded from here (Amazon Product Data): http://jmcauley.ucsd.edu/data/amazon/

The validation loss increases while training loss decreases. The best model is obtained from epoch 0. 

Attached contains the notebook and the dataset.

Can you help with identifying the issue?

Thanks
trip
[ratings_Musical_Instruments.zip](https://github.com/aws/sagemaker-python-sdk/files/2739165/ratings_Musical_Instruments.zip)

**Comment by pnpnpn** [comment]
The "validation loss increases while training loss decreases" is a common phenomenon known as overfitting when training machine learning algorithms. Object2Vec has built-in remedies to reduce overfitting:
        
1. Use dropout hyperparameter: see https://docs.aws.amazon.com/sagemaker/latest/dg/object2vec-hyperparameters.html
2. Use weight_decay hyperparameter
3. If possible, obtain more training examples
4. Finally, you can reduce the model complexity
        
I hope that helps.

**Comment by laurenyu** [comment]
closing due to inactivity. feel free to reopen if necessary.
